snow/networking/router/chain_router_test.go

```diff
@@ -325,6 +329,7 @@ func TestShutdownTimesOut(t *testing.T) {
 // Ensure that a timeout fires if we don't get a response to a request
 func TestRouterTimeout(t *testing.T) {
 	require := require.New(t)
+
```

Comment:
all the test have require in its own "block" separated by a line from the rest of the test